### PR TITLE
Token type added to the events.

### PR DESCRIPTION
**Pull request by reddyismav**

### CODE REVIEW

**Review by QEDK — APPROVED** [review]
LGTM!

**Review by rahuldamodar94 — APPROVED** [review]

**Review by jdkanani — APPROVED** [review]

File: contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol [hunk 1]
```diff
@@ -13,7 +13,11 @@ contract FxERC1155RootTunnel is FxBaseRootTunnel, Create2, ERC1155Holder {
     bytes32 public constant WITHDRAW_BATCH = keccak256("WITHDRAW_BATCH");
     bytes32 public constant MAP_TOKEN = keccak256("MAP_TOKEN");
     
-    event TokenMapped(address indexed rootToken, address indexed childToken);
+    event TokenMappedERC1155(address indexed rootToken, address indexed childToken);
+    event FxWithdrawERC1155(address indexed rootToken, address indexed childToken, address userAddress, uint256 id, uint256 amount);
+    event FxDepositERC1155(address indexed rootToken, address indexed userAddress, uint256 id, uint256 amount);
+    event FxWithdrawBatchERC1155(address indexed rootToken, address indexed childToken, address userAddress, uint256[] ids, uint256[] amounts);
+    event FxDepositBatchERC1155(address indexed rootToken, address userAddress, uint256[] ids, uint256[] amounts);
```
Comment (jdkanani) [inline contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol]: Make `userAddress` indexed?

File: contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol [hunk 2]
```diff
@@ -13,7 +13,11 @@ contract FxERC1155RootTunnel is FxBaseRootTunnel, Create2, ERC1155Holder {
     bytes32 public constant WITHDRAW_BATCH = keccak256("WITHDRAW_BATCH");
     bytes32 public constant MAP_TOKEN = keccak256("MAP_TOKEN");
     
-    event TokenMapped(address indexed rootToken, address indexed childToken);
+    event TokenMappedERC1155(address indexed rootToken, address indexed childToken);
+    event FxWithdrawERC1155(address indexed rootToken, address indexed childToken, address userAddress, uint256 id, uint256 amount);
```
Comment (jdkanani) [inline contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol]: Make `userAddress` indexed?

File: contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol [hunk 3]
```diff
@@ -13,7 +13,11 @@ contract FxERC1155RootTunnel is FxBaseRootTunnel, Create2, ERC1155Holder {
...
+    event FxDepositERC1155(address indexed rootToken, address indexed userAddress, uint256 id, uint256 amount);
```
Comment (jdkanani) [inline contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol]: Add depositor?

File: contracts/examples/erc20-transfer/FxERC20RootTunnel.sol [hunk 4]
```diff
@@ -68,6 +70,7 @@ contract FxERC20RootTunnel is FxBaseRootTunnel, Create2 {
         // DEPOSIT, encode(rootToken, depositor, user, amount, extra data)
         bytes memory message = abi.encode(DEPOSIT, abi.encode(rootToken, msg.sender, user, amount, data));
         _sendMessageToChild(message);
+        emit FxDepositERC20(rootToken, user, amount);
```
Comment (jdkanani) [inline contracts/examples/erc20-transfer/FxERC20RootTunnel.sol]: Add depositor?

File: contracts/examples/erc721-transfer/FxERC721RootTunnel.sol [hunk 5]
```diff
@@ -74,6 +76,7 @@ contract FxERC721RootTunnel is FxBaseRootTunnel, Create2, IERC721Receiver {
         // DEPOSIT, encode(rootToken, depositor, user, tokenId, extra data)
         bytes memory message = abi.encode(DEPOSIT, abi.encode(rootToken, msg.sender, user, tokenId, data));
         _sendMessageToChild(message);
+        emit FxDepositERC721(rootToken, user, tokenId);
```
Comment (jdkanani) [inline contracts/examples/erc721-transfer/FxERC721RootTunnel.sol]: Add `depositor`?

**Review by jdkanani — CHANGES_REQUESTED** [review]

File: contracts/examples/erc721-transfer/FxERC721RootTunnel.sol (hunk 5 above)
Comment (reddyismav) [reply contracts/examples/erc721-transfer/FxERC721RootTunnel.sol]: Done

File: contracts/examples/erc20-transfer/FxERC20RootTunnel.sol (hunk 4 above)
Comment (reddyismav) [reply contracts/examples/erc20-transfer/FxERC20RootTunnel.sol]: Done

File: contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol (hunk 3 above)
Comment (reddyismav) [reply contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol]: Done

File: contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol (hunk 2 above)
Comment (reddyismav) [reply contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol]: Yep! Done!

File: contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol (hunk 1 above)
Comment (reddyismav) [reply contracts/examples/erc1155-transfer/FxERC1155RootTunnel.sol]: Done!

**Review by jdkanani — APPROVED** [review]

**Pull request merged**